st2common/st2common/models/api/base.py

```diff
@@ -109,6 +111,16 @@ def decorate(f):
         def callfunction(*args, **kwargs):
             params = getattr(pecan.request, 'params', {})
 
+            # Common request information included in the log context
+            request_info = {'method': pecan.request.method, 'path': pecan.request.path,
+                            'remote_addr': pecan.request.remote_addr}
+
+            # Log the incoming request
+            values = copy.copy(request_info)
+            values['filters'] = kwargs
+            extra = prefix_with_underscore(values)
+            LOG.info('%(method)s %(path)s with filters=%(filters)s' % values, extra=extra)
```

Comment:
This way we can get rid of all the duplicated log statements in the API controllers. In addition to that, all those log messages will now include additional context such as request method, path, client ip address, etc.